frontend/src/components/account/Register.jsx

```diff
@@ -94,16 +100,23 @@ export default function Register() {
       setConfirmMsg("Confirm password must match with the password.")
       return;
     }
-    //go to the next step    
-    setActiveStep(activeStep + 1);
+
+    // Go to the profile page  
+    setActiveStep(1);
   };
 
   async function handleProfileNext() {
 
-    if (username.length === 0 || password.length === 0 || first_name.length === 0 || last_name.length === 0 || gender.length === 0 || major.length === 0 || school.length === 0 || ethnicity.length === 0 || grad.length === 0 || grad_month === 0 || grad_year === 0) {
+    if (username.length === 0 || password.length === 0 || first_name.length === 0 || last_name.length === 0 || gender.length === 0 || major.length === 0 || school.length === 0 || ethnicity.length === 0 || phone_number === undefined || phone_number.length === 0 || grad.length === 0 || grad_month === 0 || grad_year === 0) {
```

Comment:
Could we make a helper function that checks this to make this if loop condition better?

Reply:
Resolved! Created a helper function called isEmpty.